### PR TITLE
HPCC-14902 Added FindNUMA.cmake file

### PR DESCRIPTION
**Pull request by Michael-Gardner**
Signed-off-by: Michael Gardner michael.gardner@lexisnexis.com

### CODE REVIEW

**Comment by Michael-Gardner** [comment]
@xwang2713 Please review
The only thing that I'm worried about is building on windows or osx.  I think there could be an issue where the lib isn't found and I throw a fatal error.

**Comment by richardkchapman** [comment]
Builds on OSX don't need Numa - not sure about Windows

**Comment by richardkchapman** [comment]
@ghalliday Please review

File: cmake_modules/FindNUMA.cmake [hunk 1]
```diff
@@ -0,0 +1,36 @@
+################################################################################
+#    HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems®.
```
Comment (ghalliday) [inline cmake_modules/FindNUMA.cmake]: picky: Date should be 2016

**Comment by ghalliday** [comment]
@Michael-Gardner one minor issue.
Windows is fine - it isn't required, and I checked it built.

**Comment by Michael-Gardner** [comment]
I went ahead and fixed the date on the copyright notice.  Should be good for a merge @richardkchapman

**Pull request merged**